components/shared/gallery/Properties.vue

```diff
@@ -6,7 +6,9 @@
         :key="attribute[fieldKey]"
         class="column is-4">
         <div class="notification is-primary is-light">
-          <div class="is-size-7 has-text-grey-lighter is-uppercase">
+          <div
+            v-if="attribute[fieldKey] && attribute[fieldKey] !== 'undefined'"
```

Comment:
hey, why is it needed? 
`<Properties />` component is already wrapped with a `v-if="meta.attributes && meta.attributes.length"`

Reply:
Because the tag object doesn't have the property of 'trait_type' or 'display_type'.

Reply:
How did this happen? can't it be possible to leave it empty if not filled in? (I'm asking this since It can introduce spacing issue)

Reply:
It happens during the minting process. On the front-end side, it's already empty, but after process by server side, it
turn into an `undefined` string.

<img width="795" alt="image" src="https://user-images.githubusercontent.com/31397967/185940345-3cb636ed-b028-41ad-a155-2478b48d6c2f.png">

Reply:
>I'm asking this since It can introduce spacing issue

How about leaving an empty string like ' ' to keep them the same height?

Reply:
> How about leaving an empty string like ' ' to keep them the same height?

Let's try this solution

Reply:
<img width="860" alt="image" src="https://user-images.githubusercontent.com/31397967/185954985-bdddf7c9-56db-40d3-af75-b2fd50601a56.png">